syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/template-sources/simple/syndesis-extension-project/syndesis-ext-xml/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,20 @@
+<configuration>
```

Comment:
missing xsd/DTD/Doctype definition. it leads to warning in Eclipse, need to add at the top:

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE xml>

Reply:
done